README.rst

```diff
@@ -87,6 +87,48 @@ You can check payment status.
     #[Out]#              ('authCode', '042760'),
     #[Out]#              ('dttime', datetime.datetime(2016, 6, 15, 10, 45, 1))])
 
+Structured data (i.e. ``cart``, ``customer`` and ``order``) are passed to ``payment_init`` as dataclasses, for some items
+appropriate enumerations are available.
+
+.. code-block:: python
+
+    tz_cet = timezone(timedelta(hours=1))
```

Comment:
This won't work properly in summer time (+2). It should be like this:

```python
from datetime import datetime
import pytz

cet = pytz.timezone('Europe/Prague')

cet.localize(datetime(2022, 1, 12, 12, 10, 37))
datetime.datetime(2022, 1, 12, 12, 10, 37, tzinfo=<DstTzInfo 'Europe/Prague' CET+1:00:00 STD>)
# '2022-01-12T12:10:37+01:00'

cet.localize(datetime(2022, 6, 12, 12, 10, 37))
datetime.datetime(2022, 6, 12, 12, 10, 37, tzinfo=<DstTzInfo 'Europe/Prague' CEST+2:00:00 DST>)
# '2022-06-12T12:10:37+02:00'
```

Reply:
Well, this is a README example and I didn't want to complicate it with unrelated things. I am aware of the problem and that is why I used dates from winter time (CET) and named the timezone accordingly.

Anyway, I switched to unambiguous UTC in the example.

Reply:
I agree that the name "cet" can be misleading. For example, "myzone" can be used instead. In my opinion, the CET/CEST zone example is more useful than UTC, especially for less experienced developers.

Reply:
OK, I changed a README example - with standard `ZoneInfo` (3.9+ seems to be a good tradeoff for simplicity to me).